### ISSUE TITLE
[US - DEV] As MILES HARDIN, I want to see my information on a board in the workshopping page so that people can recognize me as the creator of the idea.

### ISSUE THREAD

**Issue by JulieMass** [dev-launchers/dev-launchers-platform#742]

**Comment by liwenlin664477** [comment]
Hi Julie, I think that is good for me and I just want to know what kind of block it is. Thanks!

**Comment by JulieMass** [comment]
> Hi Julie @JulieMass, I think that is good for me and I just want to know what kind of block it is. Thanks!

------------------------------------------------------

Hi @liwenlin664477 awesome 😀! thank you so much for taking this task Will! It is all yours 💪, just be aware that we had a design task dependency for @Nouri-Anouar here because our designs for the board aren't updated yet. So I would suggest that you both work together defining the missing parts and "UX/dev pair" for this task like we did before with Iris. 

What is pending to define is the following:
- how are we going to call Miles (the person who submits an idea)? should we call them "idea owner", "idea creator", another name? and based on what it is defined we would have to update the designs on this board. 

The rest are things that need to be removed from the designs like the email, etc. (please check the picture I added above in the description of the user story)

You both can talk and define this in our next weekly on Sunday with the rest of the team too. It would be perfect because you can have other points of view too and it would be more efficient. What do you think?

**Comment by JulieMass** [comment]
Ah also, I forgot: please @liwenlin664477 can you assign this task to yourself and estimate it?

**Comment by liwenlin664477** [comment]
@JulieMass Yes Julie, I just estimated it.

**Comment by Nouri-Anouar** [comment]
> > Hi Julie @JulieMass, I think that is good for me and I just want to know what kind of block it is. Thanks!
> 
> Hi @liwenlin664477 awesome 😀! thank you so much for taking this task Will! It is all yours 💪, just be aware that we had a design task dependency for @Nouri-Anouar here because our designs for the board aren't updated yet. So I would suggest that you both work together defining the missing parts and "UX/dev pair" for this task like we did before with Iris.
> 
> What is pending to define is the following:
> 
> * how are we going to call Miles (the person who submits an idea)? should we call them "idea owner", "idea creator", another name? and based on what it is defined we would have to update the designs on this board.
> 
> The rest are things that need to be removed from the designs like the email, etc. (please check the picture I added above in the description of the user story)
> 
> You both can talk and define this in our next weekly on Sunday with the rest of the team too. It would be perfect because you can have other points of view too and it would be more efficient. What do you think?

Thank is a great idea @JulieMass, I would love to work with you and @liwenlin664477 on this.
I just updated the product owner card design. looking forward to discuss this with you all today.

**Comment by Nouri-Anouar** [comment]
![image](https://user-images.githubusercontent.com/79311912/208388791-65d62769-14e8-43fd-acf1-1f9a68e48ae7.png)

**Comment by JulieMass** [comment]
Hey @liwenlin664477 I noticed this item was moved to "Review/QA" column 16 days ago. I'm a little lost here: Is this task completed? have you sent a pull request for Dave to review? because my understanding was that you both were going to meet so Dave could continue this task were you left off. 

(CC. @dbradham )

**Comment by liwenlin664477** [comment]
Yes, I remember I have already pushed my code. Let me double-check about it. If not, I will push it again. Sorry for the misunderstanding @JulieMass .

**Comment by liwenlin664477** [comment]
Yes, @JulieMass. For this issue, the code is in the Will-Li/Workshopping-page-save-button-and-creator-information branch.

**Comment by JulieMass** [comment]
> Yes, @JulieMass. For this issue, the code is in the Will-Li/Workshopping-page-save-button-and-creator-information branch.

Hi @liwenlin664477 **thank you soooo much 🙌 🙏!** Dave also mentioned you both went through this together already as well.

**Comment by dbradham** [comment]
Confirmed this is working as expected @JulieMass can this ticket be closed?
<img width="1182" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeEFGQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--96e7586f1d2c0ac360450bd35676b33171973bb1/image.png" alt="image.png" />

**Comment by dbradham** [comment]
@chadcrotchett @JulieMass I am moving this to done because it passes all of my QA tests. However, the profile picture shown here is essentially a random photo. I'm not sure if we have the ability within user profile to set the profile picture, so I don't have a way to implement or test that.

**Comment by JulieMass** [comment]
Hi @dbradham ,

I'm currently reviewing this ticket in colum "done" but I've identified a couple of issues that prevent us from closing it:

**- Mismatch between Acceptance Criteria and Staging Environment:** There's a discrepancy between what's described in the acceptance criteria and what's currently implemented in the staging environment (screenshot below).

**- Exposed PII in Global Staging:** The global staging environment appears to contain what might be personally identifiable information (PII). Sharing emails publicly is a major concern.

**Given these findings, I'm moving this ticket back to the product backlog**. We'll need to reassign it to a developer to address the mismatch and ensure PII is removed from the staging environment. 

@chadcrotchett and @dbradham , please let me know if you have any suggestions or alternative approaches we should consider.

<img width="1642" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBLzF3QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1ccc1cff79bc50d91d16800d7bc4e29fd603d402/image.png" alt="image.png" />
